webpack.config.js

```diff
@@ -95,6 +95,8 @@ if ( CALYPSO_ENV === 'desktop' || CALYPSO_ENV === 'desktop-mac-app-store' ) {
 	// no chunks or dll here, just one big file for the desktop app
 	webpackConfig.output.filename = '[name].js';
 } else {
+	webpackConfig.entry.polyfill = [ path.join( __dirname, 'client', 'lib', 'polyfill' ) ];
```

Comment:
Eugh, makes me wonder if we should just write it in plain JS and source it as a static resource from `public` or WPCOM.